docgen/parser/model.go

```diff
@@ -23,6 +23,33 @@ type File struct {
 	CustomHeaderIds bool
 }
 
+// SwaggerScheme gets scheme that you most probably want.
+// The order is https - http - wss - ws.
+// Includes "://" string if present, returns empty string if there is no valid scheme.
+func (f *File) SwaggerScheme() string {
+	if len(f.Swagger.Schemes) == 0 {
+		return ""
+	}
+	hasScheme := map[options.Swagger_SwaggerScheme]bool{}
+	for _, scheme := range f.Swagger.Schemes {
+		hasScheme[scheme] = true
+	}
+	if hasScheme[options.Swagger_HTTPS] {
+		return "https://"
```

Comment:
I feel like we could replace the `hasScheme` map with a switch case and always returns `fmt.Sprintf("://", strings.lowercase(options.Swagger_SwaggerScheme.String()))`.

Reply:
it's an array though

Reply:
that is, `f.Swagger.Schemes` is an array of more schemes, which can be `http`, `https`, `ws`, `wss`, or "unknown". 

This function is used only in `example` in the template with the `curl`, I think.